readme.md

```diff
@@ -806,6 +806,13 @@ Content about: Interviews, Portfolio Reviews, Getting a Job \
 Featured playlists: `Entrepreneurship`, `How to apply to jobs`. \
 <br/>
 
+[<img align="left" height="94px" width="94px" alt="Joma Tech channel's avatar" src="https://yt3.ggpht.com/a/AATXAJwnVxHV9cR_nofTeCXS3x2H2jRQjH3YEq8sYqpM=s176-c-k-c0x00ffffff-no-rj"/>](https://www.youtube.com/channel/UCV0qA-eDDICsRR9rPcnG7tw)
+
+[**Joma Tech**](https://www.youtube.com/c/JomaOppa/videos) [<img height="16px" width="16px" alt="Badge for verified YouTube channels" src="media/badge-verified.svg" title="Is a verified YouTube channel"/>](badges.md#verified-youtube-channel) 
```

Comment:
```suggestion
[**Joma Tech**](https://www.youtube.com/c/JomaOppa) [<img height="16px" width="16px" alt="Badge for verified YouTube channels" src="media/badge-verified.svg" title="Is a verified YouTube channel"/>](badges.md#verified-youtube-channel) \
```
Remove the "videos" parameter from the link (the link should redirect to the main section of the YouTuber) and add a backslash for layout matters.